include/ppx/application.h

```diff
@@ -315,10 +316,12 @@ class Application
     virtual void MouseDown(int32_t x, int32_t y, uint32_t buttons) {}                         // Mouse down event
     virtual void MouseUp(int32_t x, int32_t y, uint32_t buttons) {}                           // Mouse up event
     virtual void Scroll(float dx, float dy) {}                                                // Mouse wheel or touchpad scroll event
+    virtual void Knobs() {}
```

Comment:
Could you give this a more descriptive name and add a comment? Like SetupKnobs for example?